TheaterSchedule.BLL/Infrastructure/ErrorDetails.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace TheaterSchedule.Infrastructure
+{
+    public class ErrorDetails
+    {
+        public int StatusCode { get; set; }
+        public string Message { get; set; }
+
+        public override string ToString()
+        {
+            return JsonConvert.SerializeObject(this);
```

Comment:
don't mix string representation of object and serialization

ToString is mentioned to be some user friendly representation of object